internal/stores/objects.go

```diff
@@ -331,12 +332,9 @@ type (
 		// Root uniquely identifies a sector and is therefore the primary key.
 		Root consensus.Hash256 `gorm:"index;NOT NULL;type:bytes;serializer:gob"`
 
-		// Host is the key of the host that stores the sector.
-		// TODO: once we migrate the contract store over to a relational
-		// db as well, we might want to put the contract ID here and
-		// have a contract table with a mapping of contract ID to host.
-		// That makes for better migrations.
-		Host consensus.PublicKey `gorm:"index;NOT NULL;type:bytes;serializer:gob"`
+		// ContractID is the ID of the contract storing the sector.
+		ContractID types.FileContractID `gorm:"type:bytes;serializer:gob"`
```

Comment:
Did you consider a many-to-many table? The same sector can exist on multiple hosts right?

Reply:
pretty cool stuff:
https://gorm.io/docs/many_to_many.html#Many-To-Many
https://gorm.io/docs/associations.html#Association-Mode